de.metas.async/src/main/java/de/metas/async/model/validator/Main.java

```diff
@@ -57,6 +57,10 @@
  */
 public class Main extends AbstractModuleInterceptor
 {
+	public static transient String SYSCONFIG_Async_InitDelayMillis = "Async_InitDelayMillis";
```

Comment:
* Please prefix this with `de.metas.async`.
* It's important to have the UOM (millis) like you have them here
* in this case, seconds might also be sufficuently granular, but millis is also ok

Reply:
Done